height: 50
width: 50
cell_size: 0.5
DEBUG: true
